router/server.go

```diff
@@ -22,6 +24,8 @@ import (
 	"time"
 )
 
+var ConfigServiceUrl = "http://localhost:2010"
```

Comment:
You can open the config file for the config service and then read it's URL from that which means if the config changes the router knows about it immediately and we do not have to change the location in code